modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -225,9 +223,8 @@ class PersistentActor[S, M, R, E](
   }
 
   private def handle(state: InternalActorState, msg: ProcessMessage[M]): Unit = {
-    val ProcessMessageSpan = createSpan("process_message").setTag("aggregateId", aggregateId)
     context.setReceiveTimeout(Duration.Inf)
-    context.become(persistingEvents(state.withMessageProcessingSpan(ProcessMessageSpan)))
```

Comment:
got rid of .withMessageProcessingSpan for now because it was getting in my way, but I'll get back to adding more details wrt to PersistentActor in a subsequent PR